testsuite/forge/src/backend/k8s/cluster_helper.rs

```diff
@@ -310,3 +342,183 @@ pub async fn k8s_client() -> K8sClient {
+    };
+    Ok(eks_client)
+}
+
+fn create_update_nodegroup_config_request(
+    cluster_name: &str,
+    nodegroup_name: &str,
+    nodegroup_scaling_config: NodegroupScalingConfig,
+) -> UpdateNodegroupConfigRequest {
+    UpdateNodegroupConfigRequest {
+        client_request_token: None,
+        cluster_name: cluster_name.to_string(),
+        labels: None,
+        nodegroup_name: nodegroup_name.to_string(),
+        scaling_config: Some(nodegroup_scaling_config),
+    }
+}
+
+async fn submit_update_nodegroup_config_request(
+    eks_client: &EksClient,
+    cluster_name: &str,
+    nodegroup_name: &str,
+    nodegroup_scaling_config: NodegroupScalingConfig,
+) -> Result<String> {
+    let update_nodegroup_request = create_update_nodegroup_config_request(
+        cluster_name,
+        nodegroup_name,
+        nodegroup_scaling_config,
+    );
+    let update_response = eks_client
+        .update_nodegroup_config(update_nodegroup_request)
+        .await;
+    let update_id = update_response.unwrap().update.unwrap().id.unwrap();
+    println!(
+        "Created {} nodegroup update request with ID: {}",
+        nodegroup_name, update_id
+    );
+    Ok(update_id)
+}
+
+pub fn set_eks_nodegroup_size(
+    cluster_name: String,
+    num_validators: usize,
+    auth_with_k8s_env: bool,
+) -> Result<()> {
+    // https://github.com/lucdew/rusoto-example/blob/master/src/client.rs
+    // Create rusoto client through an http proxy
+    let eks_client = create_eks_client(auth_with_k8s_env).unwrap();
+    println!("Created rusoto http client");
+
+    // nodegroup scaling factors
+    let max_surge = 2; // multiplier for max size
+    let num_validators: i64 = num_validators as i64;
+    let idle_utilities_size = 5; // keep extra utilities nodes around for forge pods and monitoring
+    let validator_scaling = NodegroupScalingConfig {
+        desired_size: Some(cmp::max(num_validators * 3, 1)),
+        max_size: Some(cmp::max((num_validators * 3 + 1) * max_surge, 1)),
+        min_size: Some(cmp::max(num_validators * 3, 1)),
+    };
+    let utilities_scaling = NodegroupScalingConfig {
+        desired_size: Some(cmp::max(num_validators * 3, idle_utilities_size)),
+        max_size: Some(cmp::max(
+            num_validators * 3 * max_surge,
+            idle_utilities_size,
+        )),
+        min_size: Some(cmp::max(num_validators * 3, idle_utilities_size)),
+    };
+    let trusted_scaling = NodegroupScalingConfig {
+        desired_size: Some(num_validators),
+        max_size: Some(cmp::max(num_validators * max_surge, 1)),
+        min_size: Some(num_validators),
+    };
+
+    // submit the scaling requests
+    let rt = Runtime::new().unwrap();
+    let validators_update_id = rt
+        .block_on(submit_update_nodegroup_config_request(
+            &eks_client,
+            &cluster_name,
+            "validators",
+            validator_scaling,
+        ))
+        .unwrap();
```

Comment:
Lets return the error instead of panicking
```suggestion
        ?;
```

Reply:
There's also a couple other places in this diff that should be tweaked too.